tests/tests_integration/test_api/test_files.py

```diff
@@ -209,8 +211,37 @@ def test_upload_bytes_with_nordic_characters(self, cognite_client: CogniteClient
         external_id = "test_upload_bytes_with_nordic_characters"
 
         _ = cognite_client.files.upload_bytes(
-            content=content, name="nordic_chars.txt", external_id=external_id, overwrite=True
+            content=content,
+            name="nordic_chars.txt",
+            external_id=external_id,
+            overwrite=True,
         )
 
         retrieved_content = cognite_client.files.download_bytes(external_id=external_id)
         assert retrieved_content == content.encode("utf-8")
+
+    def test_upload_multipart(self, cognite_client: CogniteClient) -> None:
+        # Min file chunk size is 5MiB
+        content_1 = "abcde" * 1_200_000
+        content_2 = "fghij"
+
+        external_id = "test_upload_multipart"
+
+        cognite_client.config.file_transfer_timeout = 10
+
+        session = cognite_client.files.begin_multipart_upload(
+            name="test_multipart.txt",
+            parts=2,
+            external_id=external_id,
+            overwrite=True,
+        )
+
+        assert len(session.upload_urls) == 2
+
+        cognite_client.files.upload_multipart_part(session.upload_urls[0], content_1, session.file_metadata.mime_type)
+        cognite_client.files.upload_multipart_part(session.upload_urls[1], content_2, session.file_metadata.mime_type)
+
+        cognite_client.files.complete_multipart_upload(session)
+
+        retrieved_content = cognite_client.files.download_bytes(external_id=external_id)
+        assert len(retrieved_content) == 6000005
```

Comment:
It should delete the file afterwards?
Or are the files cleaned up in some other way by the python SDK integration test tooling?